management_api_app/api/routes/workspaces.py

```diff
@@ -93,20 +100,61 @@ async def create_workspace_service(workspace_create: WorkspaceServiceInCreate,
     return WorkspaceServiceIdInResponse(workspaceServiceId=workspace_service.id)
 
 
+@router.post("/workspaces/{workspace_id}/workspace-services/{service_id}/user-resources",
+             status_code=status.HTTP_202_ACCEPTED,
+             response_model=UserResourceIdInResponse, name=strings.API_CREATE_USER_RESOURCE)
+async def create_user_resource(user_resource_create: UserResourceInCreate,
+                               user_resource_repo: UserResourceRepository = Depends(
+                                   get_repository(UserResourceRepository)),
+                               user: User = Depends(get_current_user),
+                               workspace: Workspace = Depends(get_workspace_by_workspace_id_from_path),
+                               workspace_service: WorkspaceService = Depends(get_workspace_service_by_id_from_path)
+                               ) -> UserResourceIdInResponse:
+    access_service = get_access_service()
```

Comment:
should we not check here to make sure that the service is actually deployed before adding items to it? - otherwise we could have a situation where we are trying to add user resources to a workspace service that has either failed to installed or is still pending installation.

Reply:
same applies to adding workspace services to workspaces -- just didn't catch it in that PR - but we probably need to verify in both places